deps/v8/include/v8-version.h

```diff
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 4
 #define V8_MINOR_VERSION 5
 #define V8_BUILD_NUMBER 103
-#define V8_PATCH_LEVEL 43
+#define V8_PATCH_LEVEL 44
```

Comment:
IMO it is not necessary to bump the patch level on each commit – although an argument could be made that the commits be squashed. Upstream does squash when doing back-merges. In this case, each of the commit ended being slightly different from the upstream version, but as long as that is documented in the PR here, it might be okay to squash? I don't have strong feelings on this however.